### DISCUSSION TITLE
Implement support for non-integer values for BigNumber factorial

### DISCUSSION THREAD

**Discussion by sonnyk22** [josdejong/mathjs#2703] [Ideas]
When I config the library to use `BigNumber`, Unfortunately it is throwing an exception, but it works fine when it is set to `number`.  In the calculator application I am implementing, it is important for me to have it defaulted to `BigNumber` setting.

**Sample code:**
![calc1](https://cloud.githubusercontent.com/assets/6744710/13964457/2ad25e46-f040-11e5-8c03-91dec9b38e03.png)

**Result:**
![calc2](https://cloud.githubusercontent.com/assets/6744710/13964461/2d1f7d1e-f040-11e5-84ca-46375d7d2bd5.png)

**Comment by josdejong** [comment]
The BigNumber implementation of `factorial` lacks support for decimals. I will put this as a feature request and update the title of the issue accordingly.

**Comment by sonnyk22** [comment]
Thank you!

**Comment by avianey** [comment]
any update on this ?

**Comment by josdejong** [comment]
No progress yet. 

Anyone interested in picking this one up?

**Comment by thetazero** [comment]
This seems to still be an issue, I'll take it up

**Comment by josdejong** [comment]
Thanks @thetazero 👍
